src/components/commmon/ErrorBlock/ErrorBlock.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+import { P5 } from "@deskpro/deskpro-ui";
+import type { FC, JSX } from "react";
+import type { Maybe } from "../../../types";
+
+type Props = {
+  text?: Maybe<string|JSX.Element|Array<string|JSX.Element>>,
```

Comment:
This seems overly complex. I would recommend setting this to something like `texts: ReactNode[]`. Yes, the calling code now has to always pass in an `array` but it vastly reduces the complexity.

Reply:
Sounds good.